src/NATS.Client.JetStream/NATS.Client.JetStream.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <!-- NuGet Packaging -->
+    <PackageTags>pubsub;messaging</PackageTags>
+    <Description>JetStream support for NATS.Client.</Description>
+    <IsPackable>true</IsPackable>
```

Comment:
May want to flip to `false` with a comment of "change to true once ready to publish"

Otherwise if this gets into `main` before we are ready to release, and a version is incremented, then it will end up going out

Reply:
Very good point!